File: lodes/Sorting/Insertion/insertion.py
def insertion(values):
    for i in range(1, len(values)):
        for j in range(i-1, -1, -1):
            if values[j+1] < values[j]:
                tmp = values[j]
                values[j] = values[j+1]
                values[j+1] = tmp

    return values

File: lodes/Sorting/Insertion/insertion.c
void insertion(int *values, int length)
{
    for(int i = 1; i < length; i++) {
        for(int j = i - 1; j >= 0; j--) {
            if(values[j] > values[j+1]) {
                int tmp = values[j+1];
                values[j+1] = values[j];
                values[j] = tmp;
            }
        }
    }
};
